docs/slides/multi-cloud/adp/index.md

```diff
@@ -0,0 +1,634 @@
+  * Fips 140-2 (140-3) Compliance
+
+---
+# Data Breaches: The application layer
+* With cloud adoption, the traditional approach to securing customer data breaks down
+* Adversary’s are not typically breaking into a cloud datacenter and stealing physical hardware
+* They are breaking into organizations via phishing attacks, exposed networks, and supply chain attacks
+* Once inside the network they are escalating credentials and gaining privileged access to databases and systems
+
+---
+class: col-2
+# Compromised DBA Creds
+Breaches are commonly carried out via attackers who have gained escalated credentials. They were then able to bypass TDE as an example. CC numbers are exposed in plaintext
+![:scale 100%](images/slide49.png)
+
+---
+# Solution: Transform Secret Engine
+1. Vault’s Transform And Transit Secret Engines function as an Encryption-as-a-Service
+1. Developers use Vault to encrypt and decrypt data outside of Vault
+.center[![:scale 70%](images/slide50.png)]
+
+---
+# Transform (& Transit) Engine Benefits:
+* Vault’s Transform Engine provides developers with a well-architected API so they do not have to become encryption or cryptography experts
+* Vault is platform agnostic so developers can code against one API
+* It ensures approved ciphers and algorithms are used
+* It supported automated key rotation and re-wrapping
+* If an attacker manages to get access to the encrypted data, they will only see ciphertext that is useless without Vault.
+* The Transform Secret engine is Format Preserving. Thus, it does not require any changes to database structure
+  * i.e. 16 digits CCNs are encrypted as 16 digit ciphertext
+
+---
+# Transform Data Masking
+.center[![:scale 90%](images/slide50.png)]
+
+---
+# Transit Secret Engine
+.center[![:scale 90%](images/slide53.png)]
+
+---
+# Example with Encryption Enabled
+.center[![:scale 90%](images/Transit_enabled.png)]
+
+---
+class: col-2
+# Tokenization
+.smaller[
+* Non-Reversible Identification: Protect data pursuant to requirements for data irreversibility (PCI-DSS, GDPR, etc.) with strong forward secrecy
+* Integrated Metadata: Supports metadata for identifying data type and purpose
+]
+.center[![:scale 50%](images/tokenization_high_level.png)]
+
+---
+# Tokenization (Diagram)
+.center[![:scale 90%](images/tokenization_workflow.png)]
+
+---
+# Transform vs. Transit vs. Tokenization
+.center[![:scale 60%](images/vs.png)]
+
+
+---
+class: title, shelf, no-footer, fullbleed
+background-image: url(images/HashiCorp-Title-bkg.jpeg)
+
+# Lab 3:
+## Vault Transform & Transit Secret Engines
+
+---
+# Lab Environment
+In this next Lab we’ll use a web application that leverages both the Transform & Transit Secret engines to encrypt and decrypt data
+* The “HashiCups” application will leverage Vault’s API to encrypt customer CCN’s before writing them to the backend database
+* This lab will also showcase data-masking as well
+
+Lab Link: https://play.instruqt.com/hashicorp/tracks/adp-vault
```

Comment:
I don't see an adp-vault track added to this repo in your PR.  Where does the source for that live?

Reply:
The source code for that Track lives here: https://github.com/hashicorp/field-demos-vault-expand-team-transformers

Reply:
I recommend removing this link and instead writing "Will be provided by instructor."